tests/spread/plugins/catkin/snaps/ros-talker-listener/snap/snapcraft.yaml

```diff
@@ -3,6 +3,7 @@ version: '1.0'
 summary: ROS Example
 description: Contains talker/listener ROS packages and a .launch file.
 confinement: strict
+base: core18
```

Comment:
Was this part of the original backport? I don't believe it needs to be here, legacy won't do anything meaningful with it.

Reply:
This is [on my other branch too](https://github.com/artivis/snapcraft/commit/c76ec02dd2957d21bb7490f2a760753637e0a12d#diff-c1b90dabf4e9f14a0aa8d12648020edbR6) and was copied from [here](https://github.com/snapcore/snapcraft/pull/2437/files#diff-c1b90dabf4e9f14a0aa8d12648020edbR6). So i'd imagine it was initially there. In any case I'm removing it :+1:.

Reply:
Ah, yep, makes sense. Yeah, toast it here.

Reply:
Done.